module-2/backend/week_three.md

```diff
@@ -10,14 +10,56 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+
+* rails new
+
 2. What do Models generally inherit from in rails?
+
+* ApplicationRecord::Base
+
 3. What do Controllers generally inherit from in a rails project?
+
+* ApplicationController
+
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+
+* I'm not sure I understand what this question is asking. I assume we're talking about `config/routes.rb` related subject matter, however I'm not sure what the question is trying to achieve.
+
 5. What rake task is useful when looking at routes, and what information does it give you?
+
+* `rake routes` or `rails routes`
+* It allows us to see all of the path prefixes, Verbs, URI patterns, and Controller#Actions
+
 6. What is an example of a route helper? When would you use them?
+
+* It's a custom route created to provide access to a route that exists outside of the standard CRUD routes or "resources" routes.
+
+
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+
+* I've never actually used the "url suffix". However, the `_path` suffix tells rails that it should look for the prefix attached and figure out what controller and method should be accessed and used.
```

Comment:
_path helpers provide a site-root-relative path. You should probably use this most of the time. i.e. "/horses" Internal to my app
_url helpers provide an absolute path, including protocol and server name. i.e "localhost:3000/horses" Accessing from browser